ReSharper.FSharp/test/data/features/daemon/redundantAttributeSuffix/Multiple attributes 01.fs.gold

```diff
@@ -0,0 +1,12 @@
+﻿open System
+
+type FooAttribute() = inherit Attribute()
+type BarAttribute() = inherit Attribute()
+
+[<Foo|Attribute|(0); Bar|Attribute|(1)>]
```

Comment:
This case seem not to add much in terms of coverage for this analyzer since it's basically testing that analyzer dispatcher works correctly.